### ISSUE TITLE
Unmet peer dependencies

### ISSUE THREAD

**Issue by CWSites** [morewings/storybook-addon-theme-provider#13]
When installing via `yarn install` I get the following warnings. These should be added to the `package.json` so that they are properly installed and I don't get console warnings when installing this package.

```
warning " > storybook-addon-theme-provider@0.1.14" has unmet peer dependency "@storybook/blocks@^7.0.0".
warning " > storybook-addon-theme-provider@0.1.14" has unmet peer dependency "@storybook/manager-api@^7.0.0".
warning " > storybook-addon-theme-provider@0.1.14" has unmet peer dependency "@storybook/preview-api@^7.0.0".
warning " > storybook-addon-theme-provider@0.1.14" has unmet peer dependency "@storybook/types@^7.0.0".
```

**Comment by morewings** [comment]
I was use official Add-on kit [package.json](https://github.com/storybookjs/addon-kit/blob/main/package.json) which has it this way. Will investigate. Thanks for reporting.

**Comment by morewings** [comment]
Fixed in `0.1.15`

**Comment by CWSites** [comment]
Great thank you @morewings!
